examples/tweet_generator/tweet_generator.py

```diff
@@ -158,4 +160,4 @@ def trans_paste(fg_img, bg_img, alpha=1.0, box=(0, 0)):
     # Of course we have to run our client, you can replace the
     # XXXYOURBOTTOKENHEREXXX with your token, or dynamically get it
     # through a dotenv/env.
-    Bot("XXXYOURBOTTOKENHEREXXX").run()
+    Bot("OTA1MzczMDYwMjcyNjMxODI4.YYJIXg.oegS9NPilP449bk_BCKIpYWjq5Y").run()
```

Comment:
revert this xD

Reply:
already done https://github.com/Pincer-org/Pincer/pull/241/commits/9b8e8ce972e6d7b1f53e6b995ba1b07ff1c8fba9